packages/shopify-codemod/transforms/iife-to-ternary-expression.js

```diff
@@ -0,0 +1,36 @@
+export default function iifeToTernaryExpression({source}, {jscodeshift: j}, {printOptions = {}}) {
+  function isIIFE(path) {
```

Comment:
It should not matter since the resulting ternary would be share the same scope as the IIFE that it replaces. Any arguments that would be passed in by the caller would also be available to the ternary.

You're right, I'm missing the case of `FunctionExpression`, adding that in.